getpaid_payu/client.py

```diff
@@ -203,7 +201,7 @@ def cancel_order(self, order_id: str, **kwargs) -> CancellationResponse:
     @ensure_auth
     def capture(self, order_id: str, **kwargs) -> ChargeResponse:
         url = urljoin(self.api_url, f"/api/v2_1/orders/{order_id}/status")
-        data = {"orderId": order_id, "orderStatus": OrderStatus.COMPLETED}
+        # data = {"orderId": order_id, "orderStatus": OrderStatus.COMPLETED}
```

Comment:
Why is this commented?

Reply:
Flake8/pep said  `data` variable not used.

Reply:
OK, you've found an error :wink: The line below should send those data inside PUT request.

Reply:
Please double-check with PayU docs but I'm quite sure that `data` should be sent there.